assets/js/components/settings/SettingsApp.js

```diff
@@ -70,9 +70,10 @@ function SettingsApp() {
 							/>
 						</Cell>
 						<Cell size={ 12 }>
-							<Layout>
+							<Layout className="googlesitekit-layout--rounded googlesitekit-layout--transparent">
```

Comment:
Same here, maybe another prop for the transparent bit would help.

Reply:
👍